src/main/markdown/usingeclipse.md

```diff
@@ -1,153 +1,81 @@
-  style="width: 385px; height: 100px;" alt="screenshot"/>
-</div>
-
-## Create a Web Application<a id="creating"></a>
-
-To create a Web Application, select **File > New > Web Application Project** from the Eclipse menu.
-
-In the **New Web Application Project** wizard, enter a name for your project
-and a java package name, e.g., `com.mycompany.mywebapp` If you
-installed the Google App Engine SDK, the wizard
-gives you the option to use App Engine as well. For now, uncheck this
-option and click **Finish**.
-
-<div class="screenshot">
-  <img src="images/eclipse/web-app-wizard.png"
-  style="width: 366px; height: 409px;" alt="screenshot"/>
-</div>
-
-Congratulations, you now have a GWT enabled web application. The plugin has created a boilerplate project in your workspace.
-
-<h2 id="running">Run locally in Development Mode</h2>
-
-Right-click on your web application project and select **Debug As > Web
-Application** from the popup menu.
+## Reference
 
-This creates a **Web Application** launch configuration for
-you and launches it.  The web application launch configuration will start a
-local web server and GWT development mode server.
+* [GWT Eclipse Plugin Documentation](http://gwt-plugins.github.io/documentation/gwt-eclipse-plugin/Download.html)
 
-You will find a Web Application view next to the console window.
-Inside you will find the URL for the development mode server. Paste
-this URL into Firefox, Internet Explorer, Chrome, or Safari. If this is your first time
-using that browser with the development mode server, it will prompt you to install
-the GWT Developer Plugin. Follow the instructions in the browser to install.
-  
-<div class="screenshot">
-  <img src="images/myapplication-missing-plugin.png" alt="screenshot"/>
-</div>
 
-Once the browser plugin is installed, navigate to the URL again and the starter application will load in development mode.
-
-## Make a Few Changes
-
-The source code for the starter application is in the 
-
-`MyWebApp/src/`
-
-subdirectory, where MyWebApp is the name you gave to the project. You'll see two packages, 
-
-`com.mycompany.mywebapp.client` 
-
-and
-
-`com.mycompany.mywebapp.server`
-
-Inside the client package is code that will eventually be compiled to JavaScript and run as client code in the browser. The java files in the server package will be run as Java bytecode on a server.
-
-<div class="screenshot">
-  <img src="images/eclipse/web-app-src.png"
-  style="width: 350px; height: 243px;" alt="screenshot"/>
-</div>
-
-Look inside the 
-
-`MyWebApp.java` 
-
-file in the client package. Line 40 constructs the send button.
-
-```
-final Button sendButton = new Button("Send");
-```
+## Download Eclipse<a id="eclipse"></a>
+First download and install the Eclipse IDE from [Eclipse Downloads](http://www.eclipse.org/downloads/). 
 
-Change the text from "Send" to "Send to Server".
+* Choose `Eclipse IDE for Java Developers` if there is not a need for WTP or Maven integration. 
```

Comment:
Why listing Maven here? Isn't M2E bundled into the "for Java Developers" edition too? (I believe I never ever downloaded a "for EE Developers", and had M2E out of the box, and it worked great with the GPE)